website/addons/base/__init__.py

```diff
@@ -88,22 +88,12 @@ def __init__(self, short_name, full_name, owners, categories,
         self.high_max_file_size = high_max_file_size
         self.accept_extensions = accept_extensions
 
-        # Build template lookup
-        template_path = os.path.join('website', 'addons', short_name, 'templates')
-        if os.path.exists(template_path):
-            self.template_lookup = TemplateLookup(
-                directories=[
-                    template_path,
-                    settings.TEMPLATES_PATH,
-                ]
-            )
-        else:
-            self.template_lookup = None
-
         # Provide the path the the user_settings template
         self.user_settings_template = user_settings_template
         addon_user_settings_path = os.path.join(
-            template_path,
+            'website',
```

Comment:
Because you're using a relative path, this could break if the app is run from anywhere that's not the project root.

Best to use `settings.ADDON_PATH` (which is absolute).